composer.json

```diff
@@ -17,11 +17,11 @@
     ],
     "require": {
         "php": ">=7.2",
-        "twilio/sdk": "~5.16",
+        "illuminate/events": "^5.5 || ^6.0 || ^7.0",
         "illuminate/notifications": "^5.5 || ^6.0 || ^7.0",
+        "illuminate/queue": "^5.5 || ^6.0 || ^7.0",
         "illuminate/support": "^5.5 || ^6.0 || ^7.0",
-        "illuminate/events": "^5.5 || ^6.0 || ^7.0",
-        "illuminate/queue": "^5.5 || ^6.0 || ^7.0"
+        "twilio/sdk": "^6.0"
```

Comment:
There's a bunch of breaking changes in the version bump - have you looked through and tested the upgraded version?

https://github.com/twilio/twilio-php/blob/master/CHANGES.md#2020-02-19-version-600

Reply:
I saw the breaking changes and accounted for where they effected the integration tests. 

All tests were passing but I don’t have a particular app that needs or uses all of the functionality here (current use case is sms only) and certainly not every permutation of those features. 

I just tried to make sure the tests were passing and hoped I didn’t miss anything in contributor.md etc. 

It looks like we consume a versioned api from twilio and the sdk alone is what changed, since I wasn’t adding or changing existing functionality I didn’t do much testing beyond the phpunit tests. 

What is the recommended way to test every corner in a real world scenario for future reference?

Reply:
I also sort of assumed this would accompany a major bump since the sdk has breaking changes but I thought we were fortunate to not be effected by it.